src/components/data/data-utils.js

```diff
@@ -71,7 +71,7 @@ export const renderDataCell = (data, googleProject) => {
 
   const renderArray = items => {
     return _.map(([i, v]) => h(Fragment, { key: i }, [
-      renderCell(v.toString()), i < (items.length - 1) && div({ style: { marginRight: '0.5rem', color: colors.dark(0.85) } }, ',')
+      renderCell(v?.toString()), i < (items.length - 1) && div({ style: { marginRight: '0.5rem', color: colors.dark(0.85) } }, ',')
```

Comment:
the change from `v.toString()` to `v?.toString()` here isn't strictly necessary for this PR; viewing attributes worked before this change. But it felt like a good, safe change to make.

Reply:
The main guideline we try to follow whether to use optional chaining (i.e. `?` above) is…use it if data is fetched from an API and may not exist, and don't use otherwise as it can unintentionally hide issues and make code assumptions less clear.

Reply:
I have now reverted that optional chain!